gulpfile.js

```diff
@@ -14,6 +14,7 @@ const
 	sass = require('gulp-sass'),
 	sourcemaps = require('gulp-sourcemaps'),
 	fs = require('fs'),
+    sassLint = require('gulp-sass-lint'),
```

Comment:
- Wrong indentation + please, correct the alphabetical order of *fs* and *sassLint* variables.